README.md

```diff
@@ -45,32 +45,59 @@ for possible values. The default value is `warn`.
+| NGINX_AUTH_BASIC_ENCODED_HASHED_PASSWORD | string  |                                       | Base64-encoded hashed password (using httpasswd) for HTTP Basic Authentication                                                                                                                                    |
+| BEACH_NGINX_CUSTOM_METRICS_ENABLE        | boolean | no                                    | If support for a custom metrics endpoint should be enabled                                                                                                                                                        |
+| BEACH_NGINX_CUSTOM_METRICS_SOURCE_PATH   | string  | /metrics                              | Path where metrics are located                                                                                                                                                                                    |
+| BEACH_NGINX_CUSTOM_METRICS_TARGET_PORT   | integer | 8082                                  | Port at which Nginx should listen to provide the metrics for scraping                                                                                                                                             |
+| BEACH_NGINX_MODE                         | string  | Flow                                  | Either "Flow" or "Static"; this variable is going to be renamed in the future                                                                                                                                     |
+| BEACH_ASSET_PROXY_ENDPOINT               | string  |                                       | Endpoint of a cloud storage frontend to use for proxying requests to Flow persistent resources. Requires BEACH_PERSISTENT_RESOURCES_BASE_PATH to be set. Example: "https://assets.flownative.com/example-bucket/" |
+| BEACH_PERSISTENT_RESOURCES_BASE_PATH     | string  |                                       | Base path of URLs pointing to Flow persistent resources; example: "https://www.flownative.com/assets/"                                                                                                            |
+| BEACH_STATIC_RESOURCES_LIFETIME          | string  | 30d                                   | Expiration time for static resources; examples: "3600s" or "7d" or "max"                                                                                                                                          |
+| FLOW_HTTP_TRUSTED_PROXIES                | string  | 10.0.0.0/8                            | Nginx passes FLOW_HTTP_TRUSTED_PROXIES to the virtual host using the value of this variable                                                                                                                       |
+
+## Asset Proxy
+
+By default, the direct URL of an asset stored in the S3 bucket is used as 
```

Comment:
```suggestion
By default, the direct URL of an asset stored in the cloud storage is used as 
```

Does this only relate to S3 buckets? I guess not…

Reply:
well, in my example yes, but "cloud storage" as a general term is probably better.